client/components/product-card/README.md

```diff
@@ -139,3 +143,55 @@ The following props can be passed to the Product Card Options component:
   * `options.slug`: ( string ) Option slug
 * `optionsLabel`: ( string ) Label that is displayed above the list of product options
 * `selectedSlug`: ( string ) Currently selected product option slug
+
+<a name="product-card-action"></a>Product Card Action
+=======
+
+Product Card Action is a Product Card's sub-component for rendering action section. It consists of an intro
+text (optional) and a button.
+
+### How to use the `<ProductCardAction />`
+
+```jsx
+import React from 'react';
+import ProductCard from 'components/product-card';
+import ProductCardAction from 'components/product-card-action';
+
+export default class extends React.Component {
+	render() {
+		return (
+			<ProductCard
+				title={
+					<Fragment>
+						Jetpack Backup <strong>Daily</strong>
+					</Fragment>
+				}
+				subtitle="Purchased 2019-09-13"
+				description={
+					<p>
+						<strong>Looking for more?</strong> With Real-time backups:, we save as you edit and
+						you’ll get unlimited backup archives
+					</p>
+				}
+				hasManageSubscriptionLink
```

Comment:
We'll need to update this example because of the new prop `purchase` coming in in favor of `isPurchased` and `hasManageSubscriptionLink`.